src/plugins/cft/constants/zenmoney_accounts.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @author Ryadnov Andrey <me@ryadnov.ru>
+ */
+
+/**
+ * @type {{CARD: string, CHECK: string}}
+ */
+const TYPES = {
+    CARD:    'ccard',
```

Comment:
the same question about this: does it make any sense, does it give you more guarantees when you introduce set of constants that map directly to existing set of constants?

Reply:
At least this provides a kind of namespace. Do you have better proposition? It's not about guarantees it's about clarity and predictability.